configs/_base_/models/retinanet_r50_fpn.py

```diff
@@ -66,3 +66,25 @@
         score_thr=0.05,
         nms=dict(type='nms', iou_threshold=0.5),
         max_per_img=100))
+
+tta_model = dict(
```

Comment:
So the TTA config will write in the model's config instead of being an independent file? @HAOCHENYE Please take a look.